clusterloader2/pkg/test/interface.go

```diff
@@ -34,10 +34,11 @@ type OperationType int
 const (
 	// CREATE_OBJECT is create object operation.
 	CREATE_OBJECT = OperationType(0)
-	// UPDATE_OBJECT is update object operation.
-	UPDATE_OBJECT = OperationType(1)
+	// PATCH_OBJECT is update object (using patch) operation.
+	PATCH_OBJECT = OperationType(1)
 	// DELETE_OBJECT is delete object operation.
 	DELETE_OBJECT = OperationType(2)
+	// TODO(krzysied): Figure out way to add and implement object update.
```

Comment:
Could you move this TODO below the PATCH_OBJECT comment? Also maybe rephrase it as `Figure out how to implement UPDATE_OBJECT operation`

Reply:
done